blockchain/stake/log.go

```diff
@@ -12,18 +12,8 @@ import (
 // log is a logger that is initialized with no output filters.  This
 // means the package will not perform any logging by default until the caller
 // requests it.
-var log slog.Logger
-
 // The default amount of logging is none.
-func init() {
-	DisableLog()
-}
-
-// DisableLog disables all library log output.  Logging output is disabled
-// by default until UseLogger is called.
-func DisableLog() {
```

Comment:
This can't be removed as it would require a major API version bump to the module.  Please deprecate it instead like so:

```
// DisableLog disables all library log output.  Logging output is disabled
// by default until UseLogger is called.
//
// DEPRECATED.  This will be removed in the next major version bump.  
func DisableLog() {
	log = slog.Disabled
}
```

Reply:
The form used to specify something as deprecated in the stdlib is

```
// Deprecated: Reason and/or alternatives.
```

and I believe there are static analysis tools that recognize this particular comment and will warn on usage of deprecated symbols.